spec.html

```diff
@@ -13596,7 +13614,7 @@ <h2>Syntax</h2>
 
       <emu-clause id="sec-static-semantics-mv">
         <h1>Static Semantics: MV</h1>
-        <p>A numeric literal stands for a value of the Number type or the BigInt type.</p>
+        <p>A numeric literal stands for a value of the Number type or the BigInt type, as specified by the syntax-directed operation NumericValue. Determining this value involves deriving a mathematical value (MV) from the literal, according to the following rules.</p>
```

Comment:
```suggestion
        <p>A numeric literal stands for a value of the Number type or the BigInt type, as specified by the syntax-directed operation NumericValue. Determining this value involves deriving a mathematical value from the literal, according to the following rules.</p>
```

Reply:
I didn't incorporate this change. The reason for it seems unrelated to this PR's edits. I.e., it's a pre-existing usage, and this PR doesn't invalidate it or make it 'worse'. Moreover, there are other similar usages in the spec, so addressing them all in a separate PR would be more appropriate.

Reply:
I guess that's fair. In that case, I've opened #2435 where, among other things, I've gotten rid of these usages.